install.sh

```diff
@@ -1,5 +1,5 @@
 # !/bin/sh
-git checkout master
+git checkout rl_score
```

Comment:
Change back to master so that rpi doesn't get confused.
To use locally clone this script